src/In.ProjectEKA.HipService/Common/Constants.cs

```diff
@@ -0,0 +1,23 @@
+namespace In.ProjectEKA.HipService.Common
+{
+    public static class Constants
+    {
+        public static readonly string DateTimeFormat = "yyyy-MM-ddTHH:mm:ssZ";
+
+        public const string CURRENT_VERSION = "v1";
+        public const string SESSIONS = CURRENT_VERSION + "/sessions";
+        public const string PATH_CARE_CONTEXTS_DISCOVER = CURRENT_VERSION + "/care-contexts/discover";
+        public const string PATH_CONSENTS_HIP = CURRENT_VERSION + "/consents/hip/notify";
+        public const string PATH_LINKS_LINK_INIT = CURRENT_VERSION + "/links/link/init";
+        public const string PATH_LINKS_LINK_CONFIRM = CURRENT_VERSION + "/links/link/confirm";
+        public const string PATH_HEALTH_INFORMATION_HIP_REQUEST = CURRENT_VERSION + "/health-information/hip/request";
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5837743)